__init__.py

```diff
@@ -79,6 +80,12 @@ def check_platform_build(self, message):
             self.enclosure.activate_mouth_events()
         else:
             self.speak_dialog('platform.build.none')
+        # If this is a mainstream Linux, include basic version info
+        try:
+            opsys = re.sub(r'\\[a-z]{1}', ' ', open("/etc/issue").readline())
+            self.speak('On operating system: ' + opsys)
+        except Exception as e:
+            self.log.warning('/etc/issue read failed. ' + e.__class__.__name__ + ': ' + str(e))
```

Comment:
How about using `log.exception(` here which auto-adds the exception info?

Reply:
I did not know about log.exception.  I was just matching what I saw on line 60. I assume that should be replaced to match in coming supplementary commit.

Reply:
Thanks. One last thing, the `as e` parts are no longer necessary.

Reply:
Thanks for the feedback. I have removed the `as e` parts. Also threw in one last sanity filter line.